docs/database.md

```diff
@@ -6,31 +6,68 @@ MongoDB is een open source document-georiënteerde database en is geschreven in
 
 MongoDb is geschikt voor grote data het is schaalbaar en werk zonder relaties in tegenstelling tot bijvoorbeeld SQLite. Omdat we in ons project geen echte relaties nodig hebben is de keuze daarom ook makkelijk gemaakt voor MongoDB.
 
-### Voorbeeld eventuele database structuur
+## Database structuur
+
+### Collection polls
 
 ```json
 {
-  "users": {
-    "useridofmaliek": {
-      "username": "Maliek Meersschaert",
-      "password": "SOME HASH",
-      "polls": {
-        "1": {
-          "name": "Amerikaanse verkiezingen",
-          "options": {
-            "Donald Trump": 10,
-            "Hilary Clinton": 9,
-            "Gary Johnson": 2,
-            "Jill Stein": 1
-          }
-        }
-      }
+  "_id" : "1111c298aeb541d0776a9556",
```

Comment:
ik zou toch met een object werken als collection en dan: 

```js
{
  "polls": {
    "112SFE33F34SFD": {
      // info over poll
    }
  },
  "users": {
    "sdfqlmkjfds": {
      // info over user
    }
  }
}
```

Jammer genoeg is het dan een zwaardere operatie om alle polls van een persoon te krijgen. (loop over alle die in zijn `polls` veld zitten).

Dit heeft wel een voordeel dat je kan linken naar om het even welke poll zonder te weten wie de eigenaar is (pollastic.com/112SFE33F34SFD zou dan de poll zijn)